brownie/network/transaction.py

```diff
@@ -58,6 +59,13 @@ def wrapper(self: "TransactionReceipt", *args: Any, **kwargs: Any) -> Any:
     return wrapper
 
 
+class Status(IntEnum):
+    Dropped = -2
+    Pending = -1
+    Reverted = 0
```

Comment:
I feel like these two should be swapped?

Reply:
0 and 1 are standard values to represent reverted and successful in a txreceipt. Brownie has used -1 for pending since forever, modifying it would be a breaking change :grimacing: